bindings/python/parsers/python/model.cpp

```diff
@@ -28,11 +26,6 @@ namespace pinocchio
       // Get a dict for the global namespace to exec further python code with
       bp::dict globals = bp::extract<bp::dict>(main_module.attr("__dict__"));
 
-      // We need to link to the pinocchio PyWrap. We delegate the dynamic loading to the python
```

Comment:
@jcarpent  There is some workaround for BOOST 1.58. Can I remove them ?

Reply:
Yes, sure.